cli/src/commands/list.ts

```diff
@@ -24,35 +23,55 @@ export default class List extends Command {
     }),
   };
 
+  protected getListTable(isLocked: boolean) {
+    const listTable = {
+      columns: ['name', 'updatedAt', 'URL'],
+      config: {
+        name: {
+          headingTransform: () => 'APPLICATION',
+          minWidth: 25,
+        },
+        updatedAt: {
+          headingTransform: () => 'LAST UPDATED',
+          minWidth: 25,
+        },
+      },
+    };
+    if ( isLocked ) {
```

Comment:
We try not to put spaces inside our braces.  Given that you're not particularly consistent about it, I'd rather not start.
I'll see why our linter doesn't catch this.